sbt_common/display/src/test/scala/uk/ac/wellcome/display/models/DisplayWorkTest.scala

```diff
@@ -88,15 +89,23 @@ class DisplayWorkTest extends FunSpec with Matchers {
         identifiers = Nil,
         canonicalId = "hz2hrba9",
         publishers = List(
-          Agent("Henry Hare"),
-          Agent("Harriet Heron")
+          Unidentifiable(Agent("Henry Hare")),
```

Comment:
I notice we use `Unidentifiable` in a lot here. We should also test `Indentifiable` and `Indentified` values in this test suite.

Reply:
Makes sense. I added a more end to end one to `CreatorsTest` but yeah, We should have one here to.